buildconfig/Jenkins/buildscript

```diff
@@ -11,6 +11,19 @@
 ###############################################################################
 SCRIPT_DIR=$(dirname "$0")
 
+###############################################################################
+# Terminate existing Xvfb sessions
+###############################################################################
+if [[ -n ${CLEAN_XVFB} && "${CLEAN_XVFB}" == true ]]; then
+  echo "Terminating existing Xvfb sessions"
+
+  # Kill Xvfb processes
+  killall Xvfb
```

Comment:
This will fail if `Xvfb` isn't running, maybe add `-q`.

Reply:
I kept it simple with ` || true`.